paf-mvp-frontend/rollup.config.js

```diff
@@ -36,7 +35,7 @@ export default [
   defineConfig({
     input: relative('src/lib/one-key.ts'),
     output: {
-      file: getDestFolder(`/paf-lib.js`),
+      file: `${DIST}/paf-lib.js`,
```

Comment:
Change: now we always build files to the local `dist` directory, and _then_ we copy them to the demo, if in production.
This way the file is always up to date in this local `dist` directory (it was a nightmare otherwise, to get it in the audit log project - I would have needed to copy it from the demo directory!)